draft-ietf-anima-brski-prm.md

```diff
@@ -122,45 +120,44 @@ The approach defined here is agnostic with respect to the underlying enrollment
 
-* allows the application of registrar-agent credentials to establish TLS connections to the domain registrar. These registrar-agent credentials are different from the pledge's IDevID.
+* enables the usage of arbitrary transports between the pledge and the domain registrar via the registrar-agent; security is addressed at the application layer, and both IP-based and non-IP connectivity can be used between pledge and registrar-agent.
 
-The term endpoint used in the context of this document is similar to resources in CoAP {{RFC7252}} and also in HTTP {{RFC9110}}.
-It is not used to describe a device.
-Endpoints are accessible via .well-known URIs.
+* allows the application of registrar-agent credentials to establish TLS connections to the domain registrar; these are different from the IDevID of the pledge.
 
-To utilize the EST server endpoints on the domain registrar, the registrar-agent will act as client towards the domain registrar.
+The term endpoint used in the context of this document is equivalent to resource in HTTP {{RFC9110}} and CoAP {{RFC7252}}; it is not used to describe a device.
+Endpoints are accessible via Well-Known URIs {{RFC8615}}.
+For the interaction with the domain registrar, the registrar-agent will use existing BRSKI {{RFC8995}} endpoints as well as additional endpoints defined in this document.
+To utilize the EST server endpoints on the domain registrar, the registrar-agent will act as client toward the registrar.
 
 The registrar-agent also acts as a client when communicating with a pledge in responder mode.
 Here, TLS with server-side, certificate-based authentication is not directly applicable, as the pledge only possesses an IDevID certificate.
-The IDevID does not contain any anchor for which any kind of {{?RFC6125}} validation can be done.
+First, the IDevID does not contain any anchor for which any kind of {{?RFC6125}} validation can be done.
 Second, the registrar-agent may not be aware of manufacturer trust anchors to validate the IDevIDs.
 Finally, IDevIDs do not typically set Extended Key Usage (EKU) for TLS WWW Server authentication.
 
-The inability to effectively do TLS in responder mode is one reason for relying on object security.
-Another reason is the application on different transports channels, for which TLS may not be available, such as Bluetooth and NFC.
-
+The inability to effectively do TLS in responder mode is one reason for relying on object security at the application layer.
+Another reason is the support for alternative transports for which TLS may not be available, e.g., Bluetooth or NFC.
 Therefore, BRSKI-PRM relies on an additional signature wrapping of the exchanged data objects involving the registrar-agent for transport.
-For EST {{RFC7030}} the registrar then needs to do some pre-processing to verify this signature, which is not present in EST.
+For EST {{RFC7030}} the domain registrar then needs to do some pre-processing to verify this signature, which is not present in EST.
```

Comment:
The goal was to have as less as possible trust assumptions for the registrar-agent. Therefore it is not involved in the preprocessing. During the inital discussions there was a request to be able to verify, which registrar agent was actually involved in the bootstrapping. This is the only reason, why the registrar-agent has an own certificate, which is used to sign some data verified by the registrar. This may be useful for scenarios in which a service technician uses the registrar-agent in a commissioning tool in contrast to a registrar-agent co-located to a registrar.

Reply:
The draft should state here clearly that registrars require modifications for BRSKI-PRM.

Reply:
proposal to state:
_To utilize EST {{RFC7030}} for enrollment, the domain registrar must perform the pre-processing of this wrapping signature before actually using EST as defined in {{RFC7030}}._

to make this clearer.